analyzers/tests/SonarAnalyzer.UnitTest/Rules/UnusedPrivateMemberTest.cs

```diff
@@ -150,7 +150,7 @@ public class SkipMigration : Migration
 
         protected override void Up(MigrationBuilder migrationBuilder) { }
     }
-}").AddReferences(EntityFrameworkCoreReferences(Constants.NuGetLatestVersion)).Verify();
```

Comment:
will you open a ticket to bring this back to NuGet Latest?

For example, will we still ignore migrations on Net 8?

Reply:
No we will revert this PR during the migration of UT project to `net8.0`